README.md

```diff
@@ -300,12 +300,26 @@ Except for these limitations, we expect the API endpoints, the playground and an
 
 ## Advanced
 
-## Handling Authentication
+### Handling Authentication
 
-If you need to add authentication to your server,
-please reference FastAPI's [security documentation](https://fastapi.tiangolo.com/tutorial/security/)
-and [middleware documentation](https://fastapi.tiangolo.com/tutorial/middleware/).
+If you need to add authentication to your server, please read Fast API's documentation about [dependencies](https://fastapi.tiangolo.com/tutorial/dependencies/) and [security](https://fastapi.tiangolo.com/tutorial/security/).
 
+#### Using add_routes
+
+If you're using `add_routes`, you can use any of the following methods (see examples [here](https://github.com/langchain-ai/langserve/tree/main/examples/auth)):
+
+* [global dependencies](https://fastapi.tiangolo.com/tutorial/dependencies/global-dependencies/)
```

Comment:
nit: can we reference our examples instead, and then have our examples link to fast API? I think people probably don't want to read docs about fast api internals unless needed

Reply:
I re-wrote a bit -- I do think we need to include links to these at the moment since folks need to understand dependencies to implement auth with fast api patterns.